module_analysis/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+Copyright (C) 2019-Today: GRAP (<http://www.grap.coop/>)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ -->
+
+<odoo noupdate="1">
+
+    <record id="parameter_exclude_directories" model="ir.config_parameter">
+        <field name="key">module_analysis.exclude_directories</field>
+        <field name="value">lib,demo,test,tests,doc,description</field>
```

Comment:
Just a personal opinion.
Tests should be taken account in lines count by default
They ensure the right behavior of the module